dali/python/nvidia/dali/ops.py

```diff
@@ -1029,9 +1042,8 @@ def convert_batch(batch):
     @staticmethod
     def _function_wrapper_cpu(batch_processing, function, num_outputs, *dlpack_inputs):
         if batch_processing:
-            return PythonFunction.function_wrapper_batch(function, num_outputs,
-                                                         _dlpack_to_array, _dlpack_from_array,
-                                                         *dlpack_inputs)
+            return PythonFunction.function_wrapper_batch(function, num_outputs, _dlpack_to_array,
+                                                         _dlpack_from_array, *dlpack_inputs)
```

Comment:
Why reformat? The lines are longer now.

Reply:
This was done with auto formatting tool and now is consistent with other parts of the code. Generally, we either break argument lists, when the line length is reached (like here after the refactoring) or we do one arg per line. I know you could point places where we do not do that but I'm talking about the general case in our code base. And I don't think this spot justifies an exception.
Putting arbitrary number of arguments per line will be hard to maintain, I think. And with above remark, when you read a list of arguments with lines ending before the limit in our codebase, you generally assume that it is arg per line, so even though args were grouped somehow here, it makes it harder to read, at least for me it does.
I'm open to making it one arg per line, if you prefer it. Otherwise, as a matter of personal preference, I'm going to leave it as it is.